extensions/adobe/experience/adcloud/advertisement.schema.json

```diff
@@ -61,6 +61,52 @@
           "title": "Promoted Asset ID",
           "type": "string",
           "description": "Unique identifier for the underlying asset being promoted in the ad which this event is associated with."
+        },
+        "xdm:creativeID": {
+          "title": "creative id",
+          "type": "string",
+          "description": "Unique identifier for the creative which this event is associated with."
+        },
+        "xdm:keywordID": {
+          "title": "keyword id",
+          "type": "string",
+          "description": "Unique identifier for the keyword that surfer used to search , which triggered this click event"
+        },
+        "xdm:audienceID": {
+          "title": "audience id",
+          "type": "string",
+          "description": "Unique identifier for the audience segment for which the ad is targeted "
+        },
+        "xdm:adGroupID": {
+          "title": "adGroup id",
+          "type": "string",
+          "description": "Unique identifier for the adGroup which ad is displayed and triggered this click event "
+        },
+        "xdm:campaignID": {
+          "title": "campaign id",
+          "type": "string",
+          "description": "Unique identifier for the campaign that belongs to the current displayed advertisement which triggers this click event "
+        },
+        "xdm:network": {
```

Comment:
Elsewhere we use xdm:network to denote the different search networks (bing, google, yahoo jp etc) can we change this to xdm:networkType?

Reply:
My commit resolved this